docs/site/webpack.prod.config.js

```diff
@@ -26,7 +25,12 @@ module.exports = {
       {
         test: /\.svg$/,
         loader: 'file',
-        include: /assets\/images/,
+        include: /assets[\\\/]images/,
```

Comment:
We use `include: /images/,` on the development config. I think that we should be consistent.